torch/csrc/itt.cpp

```diff
@@ -0,0 +1,13 @@
+#include <torch/csrc/utils/pybind.h>
```

Comment:
I don't see why this shouldn't be inlined.

Reply:
This is to bind cpp function to python function in this file. Any suggestion how to deal with it?